ui/plugin-system/src/components/PluginEditor/plugin-editor-api.ts

```diff
@@ -66,7 +67,7 @@ export function usePluginEditor(props: UsePluginEditorProps) {
 
   // When kind changes and we haven't loaded that plugin before, we will need to enter a "pending" state so that we
   // can generate proper initial spec values that match the new plugin kind
-  const [pendingKind, setPendingKind] = useState('');
+  const [pendingKind, setPendingKind] = useState(props.defaultPanelKind || '');
```

Comment:
Implementing this here lets us leverage the loading logic that's already set up instead of refactoring it a ton or duplicating it.

Reply:
Whoa amazing

Reply:
Love this idea, I was thinking about trying to load a plugin in `panel-editor-slice`, this feels better